workflow_creation/utilities_auto_gui.md

```diff
@@ -1,96 +1,81 @@
-
-1. **location** : The URL of the service.
-2. **port** : The port of the service.
-3. **action** : Specfies where to lookUp the service using the URL.
-4. **name**: Name of the Particular Service/Application.
-
-The default values should be just fine for the rest of the fields.
-
-The tag **serviceConfiguration** specifies the application module (here webApp_UserInput.py) and the spreadsheet (here CAxManPritingParameters.xlsx) that is needed to initialize the application.
-
-### List of currently Supported Datatypes.
-
-Mapping w.r.t to the types and dimensions( as specified in *Unit of Measurements* ) provided in the SpeadSheet.
-
-1. **string** - string
-2. **decimal** - integer or floating point based on default values
-3. **3 decimals** - 3 fields having integer or floating point based on default values
-4. **6 decimals** - 6 fields havaing integer or floating point based on default values
-
-The web application is responsible for creation and validation of the fields. It also checks if the field is mandatory, minimum and maximum allowed values.
-
-## Running the Application
-
-The following steps are for using the the Generic Web Application interface for developing the CAxMan applications:
-
-1. Fork or copy the module *Web_Application_Generic*.
-2. Install the **Prerequisites**.
-3. Check if the dependencies are present in the parent path *"/PythonTools"*.
-4. Setting Up the *serverConfiguration* for the server.
-5. Setting Up the *serviceConfiguration* for the Application. This includes providing the path to the application module in the **script** tag and input file if any in **file** tag.
-6. Finally Launch the application service using the followingn command:
-```python
-python webAppInterface.py
-```
-7. The application when accessed in the CAxMan portal should display the generated GUI.
-8. On submission it passes the parameters to the workflow manager. Parameter by parameter. Consequently, you can connect different parameters with different services. Submission **is only allowed, if** all parameters are in their allowed range.
-
+Parameter Extraction for CloudiFacturing Generic User Input asynchronous Web Application
+===
+
+Web applications used for printing typically rely on end user input. In order to prevent malicious parameter inputs, workflow providers have to specify valid parameter ranges for inputs. A suitable way to allow for parameter configuration are spreadsheet files. For this purpose, this repository provides the _GenericGUIApplication_ GUI web application in _code\_examples/Python/app\_generic\_gui_. It allows for interactive parameter specification. When the application runs, it initially constructs appropriate GUI elements for the specified parameters. These GUI elements are incorporated into a web page, which will be presented to the user. The remainder of this document describes the requirements, configuration and deployment of the application.

```

Comment:
Please break the lines at 80 characters, it makes editing (and commenting pull requests) easier.

Reply:
Good point. Added additional line breaks.